src/main/java/frc/robot/commands/manipulator/shooter/Shooter.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.manipulator.shooter;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class Shooter extends Command {
+
+  public double shooterSpeed;
+  ShooterSubsystem sub;
```

Comment:
rename this variable to `shooter`, having `sub` anywhere doesn't really help information